.drone.star

```diff
@@ -178,7 +177,7 @@ def ocisIntegrationTests(parallelRuns, skipExceptParts = []):
                             "cd /drone/src/tmp/testrunner",
                             "composer self-update",
                             "composer --version",
-                            "make test-acceptance-core-api",
+                            "make test-acceptance-from-core-api",
```

Comment:
Note: the name of the "make" target was changed in ocis PR owncloud/ocis#5430

That is why it needs to be changed here.